frontend/src/metabase/admin/people/containers/EditUserModal/EditUserModal.tsx

```
@@ -19,19 +23,28 @@ export const EditUserModal = ({ onClose, params }: EditUserModalProps) => {
   const dispatch = useDispatch();
   const userId = params.userId ? parseInt(params.userId) : null;
   const { data: user, isLoading } = useGetUserQuery(userId ?? skipToken);
-
+  const [updateUser] = useUpdateUserMutation();
   const initialValues = useMemo(() => getInitialValues(user), [user]);
 
-  const handleSubmit = async (newValues: Partial<UserType>) => {
-    // first name and last name keys need to be present so they can
-    // potentially be removed
-    const submitValues = {
+  const handleSubmit = async (newValues: Partial<User>) => {
+    if (userId == null) {
+      return;
```

should we raise an error instead?